src/components/materialization/OnIncompatibleSchemaChange/Form.tsx

```diff
@@ -59,7 +65,7 @@ export default function Form() {
 
     return (
         <IncompatibleSchemaChangeForm
-            currentSetting={currentSetting}
+            currentSetting={currentSetting ?? onIncompatibleSchemaChange}
```

Comment:
This allows us to see the drafted version when it is there and then fallback to whatever is in the store. Useful for `create` before a `draft` is made.